docs/deploy-development-rp.md

```diff
@@ -189,16 +189,17 @@
 ## Debugging
 
 * SSH to the bootstrap node:
-
+> __NOTE:__ If you have a password-based `sudo` command, you must first authenticate before running `sudo` in the background
+> __NOTE:__ The `openssl` binary is needed to run this.  Ensure you have it installed.  
```

Comment:
I think openssl *and* openvpn are both undocumented prerequisites.  That info would go better in docs/prepare-a-shared-rp-development-environment.md though please

Reply:
It's actually referenced at the top of [this document](https://github.com/Azure/ARO-RP/blob/master/docs/deploy-development-rp.md#prerequisites) (deploy-development-rp.md).  I will add it up there.  Do we want to also add those prerequisite steps into docs/prepare-a-shared-rp-development-environment.md?